packages/tx/src/types.ts

```diff
@@ -197,6 +216,32 @@ export interface FeeMarketEIP1559TxData extends AccessListEIP2930TxData {
   maxFeePerGas?: BigIntLike
 }
 
+/**
+ * {@link BlobEIP4844Transaction} data.
+ */
+export interface BlobEIP4844TxData extends FeeMarketEIP1559TxData {
+  /**
+   * The versioned hashes used to validate the blobs attached to a transaction
+   */
+  versionedHashes?: Buffer[]
+  /**
+   * The maximum fee per data gas paid for the transaction
+   */
+  maxFeePerDataGas?: bigint
+  /**
+   * The blobs associated with a transaction
+   */
+  blobs?: Buffer[]
+  /**
+   * The KZG commitments corresponding to the versioned hashes for each blob
+   */
+  kzgCommitments?: Buffer[]
+  /**
+   * The aggregate KZG proof associated with the transaction
+   */
+  kzgProof?: Buffer
+}
+
```

Comment:
Do we need to include `gasPrice` here since `BlobEIP4844TxData` extends `FeeMarketEIP1559TxData`?

Reply:
No, 4844 also has no `gasPrice`, right?

Reply:
Correct.  I just wanted to validate since the `BlobEIP4844TxData` type just extends the 1559 one which already has `gasPrice?: never` so assumed that this would share the same value.